dir.targets

```diff
@@ -60,6 +60,8 @@
       <PackageFileRefPath Condition="'$(IsNETCoreAppRef)' == 'true'">$(NETCoreAppPackageRefPath)</PackageFileRefPath>
       <PackageFileRuntimePath>$(NETCoreAppPackageRuntimePath)</PackageFileRuntimePath>
       <RuntimePath Condition="'$(BinPlaceNETCoreAppPackage)' == 'true'">$(NETCoreAppPackageRuntimePath)\..\runtime</RuntimePath>
+      <!-- enable trimming for any project that's part of the shared framework -->
+      <SetProperties Condition="'$(IsRuntimeAssembly)' == 'true'">ILLinkTrimAssembly=true</SetProperties>
```

Comment:
IsRuntimeAssembly doesn't equate to part of the shared framework that equates to all libraries. 

Is the goal to enable this by default or should this be conditioned? Even if it is enabled by default we should have an opt-out.